config/owasp/suppressions.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress until="2030-01-01">
+    <notes><![CDATA[
+     Suppressing as it's a false positive (see: https://pivotal.io/security/cve-2018-1258)
+   ]]></notes>
+    <gav regex="true">^org\.springframework\.security:spring-security-crypto:5.2.[0-2].RELEASE</gav>
+    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    <cve>CVE-2018-1258</cve>
+  </suppress>
+
+  <suppress until="2020-01-31">
```

Comment:
Is this suppressions still required?

Reply:
I checked the NVD and both of the CVE are in Assigned status. so, kept it in the file